src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac_de.properties

```diff
@@ -60,6 +60,8 @@ javac.opt.target=Generiert Klassendateien, die sich für das angegebene Java SE-
 javac.opt.release=Kompiliert für das angegebene Java SE-Release.\nUnterstützte Releases: \n    {0}
 javac.opt.source=Liefert Quellkompatibilität mit dem angegebenen Release von Java SE.\nUnterstützte Releases: \n    {0}
 javac.opt.Werror=Kompilierung beenden, wenn Warnungen auftreten
+javac.opt.arg.Werror=<Schlüssel>(,<Schlüssel>)*
```

Comment:
This value is translated for `de`, but not for `ja` or `zh_CN`. It would be ideal for to have consistency.